src/views/TimeOutWrapper/index.tsx

```diff
@@ -104,6 +108,7 @@ const TimeOutWrapper = ({ children }: TimeOutWrapperProps) => {
         tokenExpiresIn > 0 &&
         tokenExpiresIn < fiveMinutes
       ) {
+        setTimeRemainingArr(formatSessionTimeRemaining(tokenExpiresIn));
```

Comment:
The initial state is 0 seconds so we need to set the message before the modal opens